docs/api/search/pif/query/core/PropertyQuery.md

```diff
@@ -0,0 +1,13 @@
+## Description
+
+A PropertyQuery object contains information about a desired property.
+
+## Fields
+
+Field name | Value type | Default | Description
+-----------|------------|---------|------------
+`name` | [`FieldOperation`](api/search/pif/query/core/FieldOperation) |  | Acceptable names of the property as a field operation |
```

Comment:
I don't really get these either.  I assume `FieldOperation` could, at some point, contain more than `filter` and `extract_as`?